flink-connector-base/src/main/java/com/ververica/cdc/connectors/base/source/dialect/SnapshotEventDialect.java

```diff
@@ -0,0 +1,7 @@
+package com.ververica.cdc.connectors.base.source.dialect;
+
+/**
+ * Created by yexianxun@corp.netease.com on 2022/1/7.
+ */
+public abstract class SnapshotEventDialect implements Dialect {
```

Comment:
I think the `SnapshotEventDialect` and `StreamingEventDialect ` should have different behaviors, but they need to implements same methods of `Dialect`.